src/runtime/Makefile

```diff
@@ -65,7 +66,7 @@ BINLIST += $(TARGET)
 DESTDIR ?= /
 
 ifeq ($(PREFIX),)
-PREFIX        := /usr
+PREFIX        ?= /usr
```

Comment:
@ryansavino, why do we change this?

Reply:
> @ryansavino, why do we change this?

We had wanted to be able to use this make system to generate a kata config for kata-deploy use, where the prefix might be e.g.  /opt/kata/ or /opt/confidential-containers/, and it seemed convenient to pass these prefix values as a make variable for flexibility.

Reply:
So, this check is basically `if $PREFIX == '' then PREFIX=/usr`.

https://github.com/kata-containers/kata-containers/pull/5013/files/9d4ffb92c2ff15f6377b150f2fb17ffa8e1f2641#diff-2ba0cb0d4a795ecfd09b767b2c67bbaac58abaecc743ca4c1281966d940f699fL71 is the part that **already** correctly sets up the prefix to `/opt/kata` or `/opt/confidential-containers`.

All in all, this change is not needed.

Reply:
Okay, I will revert this.